Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3170,6 +3222,126 @@ def run_remote_script_command(client: Client, args: dict) -> CommandResults:
         raw_response=run_remote_script)
 
 
+def get_remote_script_status(client: Client, args: dict) -> CommandResults:
+    """
+    Get the Satus of the Remote Script Tasks
+    """
+    headers = ["id", "createdAt", "description", "statusDescription", "parentTaskId", "accountId",
+               "accountName", "agentId", "agentIsActive", "agentOsType", "initiatedBy", "initiatedById"]
+    remote_script_statuses = client.get_remote_script_status_request(**args)
+
+    return CommandResults(
+        readable_output=tableToMarkdown("SentinelOne - Get Remote Scripts Tasks Status",
+                                        remote_script_statuses, headers=headers, removeNull=True),
+        outputs_prefix="SentinelOne.GetRemoteScript",
+        outputs=remote_script_statuses,
+        raw_response=remote_script_statuses)
+
+
+def get_remote_script_results(client: Client, args: dict) -> list[CommandResults]:
+    """
+    Get the remote script results
+    """
+    context_entries = []
+    headers = ["taskId", "fileName"]
+    # Get arguments
+    computer_names = argToList(args.get("computerNames"))
+    task_ids = argToList(args.get("taskIds"))
+    results = client.get_remote_script_results_request(computer_names, task_ids)
+    files = []
```

Comment:
Lets have a more informative name
```suggestion
    file_results = []
```